go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/feature/dynamodb/attributevalue v1.18.4
 	github.com/go-redsync/redsync/v4 v4.13.0
 	github.com/google/uuid v1.6.0
-	github.com/hashicorp/consul/api v1.31.1
+	github.com/hashicorp/consul/api v1.31.2
```

Comment:
With this change is `replace github.com/armon/go-metrics => github.com/hashicorp/go-metrics v0.4.1` still needed?

Reply:
I've tried it locally and it seems like it's still needed.